tchannel/serializer/raw.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+from tchannel.schemes import RAW
+
+
+class RawSerializer(object):
+    def __init__(self):
```

Comment:
@junchaowu I put a note elsewhere on how to fix - search "This is only used in deserialize_body"